### ISSUE TITLE
No folding of parenth_form expressions without commas

### ISSUE THREAD

**Issue by tysonclugg** [microsoft/pylance-release#3085]
Issue Type: <b>Bug</b>

1. Create a file named `example_parenth_forms.py` with the following content:
```python
parenth_form_without_comma = (
    "This is an expression list parenth_form without comma "
    "yielding a single expression made up by the expression list "
    "as defined in the Python 3 Documentation "
    "» The Python Language Reference "
    "» 6. Expressions "
    "» 6.2.3. Parenthesized forms "
    "which can be viewed at "
    "https://docs.python.org/3/reference/expressions.html#parenthesized-forms"
)

assert isinstance(parenth_form_without_comma, str), (
    "This is a single expression parenth_form without comma yielding a string"
)

parenth_form_with_comma = (
    "This is a single expression parenth_form with comma yielding a tuple",
)

assert isinstance(parenth_form_with_comma, tuple), (
    "This is a %s %s single expression parenth_form without comma" %
    ("more", "complex")
)
```

2. Open the file and observe how only the `parenth_form_with_comma` expression can be folded, none of the parenth_form expressions without commas can be folded.

Extension version: 2022.7.40
VS Code version: Code 1.69.2 (3b889b090b5ad5793f524b5d1d39fda662b96a2a, 2022-07-18T16:14:10.636Z)
OS version: Linux x64 5.15.0-41-generic
Restricted Mode: Yes

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|11th Gen Intel(R) Core(TM) i7-1185G7 @ 3.00GHz (8 x 1700)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: disabled_software<br>video_encode: disabled_software<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|1, 1, 1|
|Memory (System)|15.32GB (0.65GB free)|
|Process Argv|--unity-launch --crash-reporter-id fd6635a6-0561-48ee-9a54-e81fa01ab5c4|
|Screen Reader|no|
|VM|0%|
|DESKTOP_SESSION|gnome-xorg|
|XDG_CURRENT_DESKTOP|GNOME|
|XDG_SESSION_DESKTOP|gnome-xorg|
|XDG_SESSION_TYPE|x11|
</details>

**Comment by tysonclugg** [comment]
I've just tried the pre-release Pylance extension version `v2022.7.41` and the issue remains.  I hope you find the text in my example file useful to help resolve these issues. :slightly_smiling_face: 

This may also be related to #1768, #2333, #2542, and #2749.

**Comment by bschnurr** [comment]
This issue has been fixed in version 2022.8.41, which we've just released. You can find the changelog here: [CHANGELOG.md](https://github.com/microsoft/pylance-release/blob/main/CHANGELOG.md#2022841-24-august-2022-prerelease)

**Comment by tysonclugg** [comment]
Tested in [2022.8.41 (24 August 2022) PreRelease](https://github.com/microsoft/pylance-release/blob/main/CHANGELOG.md#2022841-24-august-2022-prerelease) and confirmed fixed.  Thanks!
